astropy/wcs/wcs.py

```diff
@@ -52,13 +52,134 @@
     AstropyWarning,
 )
 
-from . import _wcs, docstrings
+from . import docstrings
+from ._wcs import (
+    PRJ_CODES,
+    PRJ_CONIC,
+    PRJ_CONVENTIONAL,
+    PRJ_CYLINDRICAL,
+    PRJ_HEALPIX,
+    PRJ_POLYCONIC,
+    PRJ_PSEUDOCYLINDRICAL,
+    PRJ_PVN,
+    PRJ_QUADCUBE,
+    PRJ_ZENITHAL,
+    WCSCOMPARE_ANCILLARY,
+    WCSCOMPARE_CRPIX,
+    WCSCOMPARE_TILING,
+    WCSHDO_EFMT,
+    WCSHDO_P12,
+    WCSHDO_P13,
+    WCSHDO_P14,
+    WCSHDO_P15,
+    WCSHDO_P16,
+    WCSHDO_P17,
```

Comment:
I just wonder if it would make sense to add a module with all WCSLIB constants in a module `_wcs.wcslib_constants` and then use wildcard imports. But it would be more work and maybe, in the end, not as helpful as I think now.